src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -821,6 +970,12 @@ public boolean isCourseArchived(CourseAttributes course, InstructorAttributes in
         return result;
     }
     
+    /**
+     * Returns a list of courseIds for all archived courses for all instructors.
```

Comment:
We want to avoid duplication within the comment, between the method description and the `@return` statement. I don't mind if we keep the description and omit the `@return` or the other way around.